boards/AMD-Xilinx/Kria-KV260.md

```diff
@@ -0,0 +1,34 @@
+# AMD-Xilinx Kria KV260 SoC Board
+
+Product page: https://www.xilinx.com/products/som/kria/kv260-vision-starter-kit.html
+
+Arm IP:
+- [Cortex-A53](/ip/cortex-a.md) (x4)
+- [Cortex-R5F](/ip/cortex-r.md) (x2)
+- [Mali-400](/ip/mali.md) (x1)
+- [AMBA AXI4 Interconnect](/board-of-boards/ip/)
+
+
+Use Cases:
+- [AI/ML](/use-cases/ai-ml.md#embedded)
+- [Real-time Industrial Control](/use-cases/)

```

Comment:
This should link to a new use-case file, or create sub-sections of `/use-cases/iot.md` for "Industrial/Real-time"

Reply:
The KV260 should also be listed in the Robotics use case. It can be used for real time vision processing.